homeassistant/components/lifx/select.py

```diff
@@ -1,24 +1,40 @@
 """Select sensor entities for LIFX integration."""
 from __future__ import annotations
 
+from aiolifx_themes.themes import ThemeLibrary
+
 from homeassistant.components.select import SelectEntity, SelectEntityDescription
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant, callback
 from homeassistant.helpers.entity import EntityCategory
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
-from .const import DOMAIN, INFRARED_BRIGHTNESS, INFRARED_BRIGHTNESS_VALUES_MAP
+from .const import (
+    ATTR_THEME,
+    DOMAIN,
+    INFRARED_BRIGHTNESS,
+    INFRARED_BRIGHTNESS_VALUES_MAP,
+)
 from .coordinator import LIFXUpdateCoordinator
 from .entity import LIFXEntity
 from .util import lifx_features
 
+THEME_NAMES = [theme_name.lower() for theme_name in ThemeLibrary().themes]
+
 INFRARED_BRIGHTNESS_ENTITY = SelectEntityDescription(
     key=INFRARED_BRIGHTNESS,
     name="Infrared brightness",
     entity_category=EntityCategory.CONFIG,
     options=list(INFRARED_BRIGHTNESS_VALUES_MAP.values()),
 )
 
+THEME_ENTITY = SelectEntityDescription(
+    key=ATTR_THEME,
+    name="Theme",
+    entity_category=EntityCategory.CONFIG,
+    options=THEME_NAMES,
```

Comment:
Should we make these title case to be consistant with the rest of the entities?

<img width="674" alt="Screen Shot 2022-10-23 at 11 20 42 AM" src="https://user-images.githubusercontent.com/663432/197403601-9b384762-838a-4508-a9f7-969c4b40a16c.png">
<img width="528" alt="Screen Shot 2022-10-23 at 11 21 05 AM" src="https://user-images.githubusercontent.com/663432/197403637-15341b16-5d53-486e-9b07-7d2c3a106fd9.png">

Reply:
I'm a bit on the fence about that though since its probably easier to automate if we don't